repo/packages/K/kafka/0/config.json

```diff
@@ -1,6 +1,35 @@
 {
-  "type": "object",
-  "properties": {},
-  "additionalProperties": false,
-  "required": []
+    "type": "object",
+    "properties": {
```

Comment:
Namespacing is not required but it is nice because users can store uber user options files that are specialized for their datacenters.